students/WWhite/lesson04/trigrams.py

```diff
@@ -0,0 +1,48 @@
+import string, random
+
+file = open('sherlock_small.txt', 'r')
+file_text = file.read().lower()
+file.close()
+
+translator = str.maketrans(string.punctuation, ' '*len(string.punctuation))
+
+new_file_text = file_text.translate(translator).split()
+
+
+def trigrams(list):
```

Comment:
This variable name (`list`) is overwriting the Python built-in, I would suggest naming it something different and specific to your program